lib/parsers/index.ts

```diff
@@ -68,6 +68,9 @@ export function getTopLevelDeps(targetFile: ManifestFile, includeDev: boolean):
   });
 
   for (const [name, version] of dependenciesIterator) {
+    if (/^file:/.test(version)) {
+      continue;
```

Comment:
Could we not keep it but not show the transitive `dependencies`? So we can say yep we saw this dep but we can't peek inside right now so it would be top level dep only with version exactly as it was given to us. Seems nicer than throwing it away

Reply:
Very good point, I changed the approach, that makes more sense :-)